### DISCUSSION TITLE
Flower Season Brainstorm 🌷

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#3298] [Art]
## Context 🌼

The purpose of this discussion is to **brainstorm** about the Flower Season 🌷

All items, wearables and collectibles are going to be related to flowers, bees, honey and rubies. 

If you want to see how the brainstorming began, you can check these links:

**Brainstorming begins here:** https://www.twitch.tv/videos/2036690319
**Brainstorming continues here:** https://www.twitch.tv/videos/2036188561
**More information from devs here:** https://www.twitch.tv/videos/2036684848

## Examples 🧠

Here's a list of items nedeed mostly every season:

- Ticket 
- Banner 
- New resource (Flowers/Honey/Ruby)
- Fish  
- Mutant Chicken 
- Seasonal Decorations  
- SFTs related to the theme
- Wearables

**Related to these topics, what do you want to see in game?** 👀

This is how you can let us know:

- **Share options** 
Some community members think that these items may fit the season:

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/fc3ba584-7eea-42fd-967c-d5fd411e88e2)

- **Fun Facts are welcome!**
You can share fun facts related to the themes.

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/99182fad-1e53-4375-be61-c72cf27cbb2c)

- **Images**
Sometimes we just need an image to get inspiration

![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/b68455f1-9895-4121-a8e0-9ba71288cf14)

## Warnings 🚨

We'd like to bring your attention to a few important considerations:

- Be aware that any design presented here is subject to potential changes.
- Sharing ideas here won't automatically result in any personal gains for any contributor.
- Your wonderful ideas, while greatly appreciated, may not necessarily find their way into the game.
- This isn't an invitation for artists to start working on any suggestions. The design process will be handled by our Art Team.

We're eager to see the ideas you bring to the table. 
Thank you for being an essential part of our creative journey! 🌟🎨

**Comment by jojothe2nd** [comment]
- Potted Sunflowers, maybe cute or different pots or flowerbeds that they are in.
- New fences, like nice white ones, or with flower designs on them
- Mutant chicken bee, like a giant bee with a chicken beak
- Rose bushes, a different version of the current bush decoration.
- big dripping honey pot decoration
- tea party set decorations, like a cute little tea set on a table and chairs
